File: SelectionSort.py
def selectionSort(array):
    #outer loop
    n=len(array)
    for i in range(n):
        min=array[i]
        index=i
        for j in range(i+1,n):
            if array[j]<min:
                min=array[j]
                index=j
        #swapping
        array[i],array[index]=array[index],array[i]
    return array

array=[5,6,1,2,4]
print(selectionSort(array))

File: SelectionSort.js
function selectionSort(array)
{
    n=array.length
    //outer loop
    for(i=0;i<n;i++)
    {
        min=array[i]
        index=i
        //inner loop
        for(j=i+1;j<n;j++)
        {
            if(array[j]<min)
            {
                min=array[j]
                index=j 
            }
            
        }
        //swapping
        temp=array[i]
        array[i]=min 
        array[index]=temp
    }
    return array 

}
array=[2,5,8,1,0,8]
console.log(selectionSort(array))